curvefs/src/client/warmup/warmup_manager.cpp

```diff
@@ -302,7 +297,26 @@ void WarmupManagerS3Impl::FetchChildDentry(fuse_ino_t key, fuse_ino_t ino) {
             };
             AddFetchDentryTask(key, task);
             VLOG(9) << "FetchChildDentry: " << dentry.inodeid();
-        } else if (FsFileType::TYPE_SYM_LINK == dentry.type()) {  // need todo
+        } else if (FsFileType::TYPE_SYM_LINK == dentry.type()) {
```

Comment:
The logic of functions `FetchDentry` and `FetchChildDentry` is very similar, so you can consider abstracting the common parts.

Reply:
> The logic of functions `FetchDentry` and `FetchChildDentry` is very similar, so you can consider abstracting the common parts.

agree with you anymore @wuhongsong